sql/hive-thriftserver/src/main/scala/org/apache/spark/sql/hive/thriftserver/SparkSQLCLIDriver.scala

```diff
@@ -271,15 +263,6 @@ private[hive] object SparkSQLCLIDriver extends Logging {
       }
     }
 
-    // TODO: missing
```

Comment:
I prefer to wait for  @steveloughran to give a suggestion, because the code that assigns a value to `transport` is commented out by  @steveloughran. I don't quite understand the background of this commented-out code, nor did I find specific explanations from the original pr.